tutorials/creating_ssl_certificates.md

```diff
@@ -30,5 +30,7 @@ You can add the command below to that crontab. For advanced users, we suggest in
 which provides more options, and is much more powerful than certbot.
 
 ``` text
+service nginx stop
```

Comment:
Please remember that not everybody uses nginx. Rather add a warning that tells people that they might need to turn off their webserver to get certificates. The syntax for warnings is:
```
::: warning
Some warning text.
:::
```

Reply:
Yes that would also work, it's still a good idea in my opinion to give an example on how it could look with apache and nginx.